src/main/java/com/arpnetworking/metrics/mad/Main.java

```diff
@@ -250,14 +250,15 @@ private Injector launchGuice(final ActorSystem actorSystem) {
         }
 
         // Instantiate the metrics factory
+        final String sinkHost = "0.0.0.0".equals(_configuration.getHttpHost()) ? "localhost" : _configuration.getHttpHost();
```

Comment:
Fair enough. The other alternative is to allow it to be configurable separately. I think this is fine for now.